android/app/src/main/java/org/fossasia/openevent/activities/LocationActivity.java

```diff
@@ -145,6 +145,14 @@ public boolean onCreateOptionsMenu(Menu menu) {
         searchView.setQuery(searchText, false);
         return true;
     }
+    @Override
+    public void onBackPressed(){
+        if((mSessions.isEmpty())){
+            noSessionsView.setVisibility(View.VISIBLE);
+        }
```

Comment:
Yes, @Shailesh351 It is visible. This is being handled by the super.onbackpressed()